### DISCUSSION TITLE
Interaction with system map Infrastructure - power systems

### DISCUSSION THREAD

**Discussion by pgmzeta** [stonepillars/goon-flock#5] [Ideas]
My thought on flock is that conversion should have varying effects on station equipment as they adapt to this new environment. Here are some really rough ideas for how that might interact with just station power systems.

APCs
====
* Need specialized power conversion for relays to activate/charge?
* Could empower buildings/drones in the APC area / circuit
* Could have a defensive discharge (auto-targeted at enemies; like a hotwired engine zap)
* Crew retake: Destroy and rebuild the APC

SMES
====
* Flat or percentage bonus to APC boosts?
  * If solars are unwatched, you might get one or two "free" SMES, so maybe this could scale with count
* Crew retake: Cut power cables and re-establish power to the SMES to reset?
  * i.e. the gnesis is "feeding" off of the power, so if the power goes away so does the gnesis.

PTL
====
* This beams power off-station, so maybe this could serve as a capturable relay?

TEG
==== 
* Should convert to a gnesis-plated TEG!
* Crew retake:
  * Take the semiconductor out
  * Re-plate it? Might have to muck around in matsci to do this. 
    * TEG semi is unique, so we can do one-off affordances
* Reduces the amount of time it takes for relays to charge?
  * flock's auth disk :)

Vent Capture
====
* Belt SMES takeover makes drones go faster?
* Hotspots (?????????)

Singularity
====
* Not sure! Like the singulo, i am a void of nothing here.

Solars
====
* There are lots of them bunched up, and often not watched, so even a small boost would stack up quick

**Comment by stonepillars** [comment]
> # Singularity
> * Not sure! Like the singulo, i am a void of nothing here.

I see no reason why the radiation collectors couldn't be hijacked!
